profiles/nair0lf32.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/35136136/175789287-a0796936-1bf8-443c-8adc-f56c4ebb95bc.png",
+    "issueId": 1207,
+    "name": "EDEMESSI Florian",
+    "username": "nair0lf32"
```

Comment:
Looks like the indentation here is wrong.
```suggestion
  "image": "https://user-images.githubusercontent.com/35136136/175789287-a0796936-1bf8-443c-8adc-f56c4ebb95bc.png",
  "issueId": 1207,
  "name": "EDEMESSI Florian",
  "username": "nair0lf32"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient